### ISSUE TITLE
Errors in isce2 image

### ISSUE THREAD

**Issue by grallewellyn** [MAAP-Project/ZenHub#1022]
Revive and correct the isce2 image

- This commit might be the last time isce2 was actually importing from the isce2 docker https://github.com/MAAP-Project/maap-workspaces/commit/05e764703fdaf16b3319ff3d6225a6773221cc02 

Can test the installation by calling any of these functions in the [applications](https://github.com/isce-framework/isce2/tree/main/applications) folder
```
isceApp.py -h 
TopsApp.py -h
stripmapApp.py -h
```

CC @gchang @marjo-luc @sujen1412 @wildintellect

**Comment by grallewellyn** [comment]
Image with isce2 working was deployed for the user
Code here: https://github.com/MAAP-Project/maap-workspaces/tree/bug/fix-isce2-1022
Waiting on user feedback before publishing corrected isce2 image

**Comment by wildintellect** [comment]
@grallewellyn can you supply instructions on how to test the image? I assume you've deployed it to DIT for testing?

**Comment by grallewellyn** [comment]
Yes, you can modify the devfile container image to be 'mas.dit.maap-project.org/root/maap-workspaces/jupyterlab/isce2:isce2-updated' for testing
I also shared an ops workspace with Narayanarao so he can test

**Comment by grallewellyn** [comment]
Narayanarao confirmed this is working for him and hasn't broke up any problems with the image except requesting that the version of python be downgraded to 3.10 (which I created a new image for) 

User states that he doesn't believe there is anything wrong with the isce2 installation
